src/Symfony/Component/VarDumper/Cloner/VarCloner.php

```diff
@@ -36,14 +41,15 @@ protected function doClone($var)
         $maxItems = $this->maxItems;
         $maxString = $this->maxString;
         $cookie = (object) array();     // Unique object used to detect hard references
-        $gid = uniqid(mt_rand(), true); // Unique string used to detect the special $GLOBALS variable
         $a = null;                      // Array cast for nested structures
         $stub = null;                   // Stub capturing the main properties of an original item value
                                         // or null if the original value is used directly
 
         if (!self::$hashMask) {
+            self::$gid = uniqid(mt_rand(), true); // Unique string used to detect the special $GLOBALS variable
```

Comment:
a fixed value would allow people to know at it and start using it (I'm sure someone will if we allow this, people are creative...) A unique random value is free from any pb, nothing to fix nor enhance here to me.